app/utils/Strings.scala

```diff
@@ -23,6 +23,8 @@ import scala.util.Random
 object Strings {
 
   def between1And9: Int = Gen.choose(1, 9).sample.getOrElse(1)
```

Comment:
Rather than having multiple of these between functions, why don't we supply the two numbers as parameters e.g. numberBetween(min: Int, max: Int) = Gen.choose(min, max).sample.getOrElse(min)

Reply:
That said, this is something that's already there. I have my misgivings about this `Strings` object, but we should leave this alone for now because it's going to end up cascading to other areas it's used.

Reply:
Oookay fine! :') ...not sure when it was introduced but this type of code hurts my heart haha